android/app/src/main/res/values-ro/strings.xml

```diff
@@ -1918,4 +1921,8 @@
 	<string name="type.amenity.events_venue">Locul de desfășurare a evenimentelor</string>
 	<string name="type.shop.auction">Licitație</string>
 	<string name="type.shop.collector">Colecții</string>
+	<string name="type.self_service.yes">Self-service, dar nu este specificat</string>
```

Comment:
Since in all the other strings we translated "self-service", let's also do it here: "Autoservire, dar nu este specificat".

Reply:
Done ^^. Thanks for your help ^^.